.github/workflows/ci.yml

```diff
@@ -110,48 +98,36 @@ jobs:
       fail-fast: false
       matrix:
         runs-on:
+          - ubuntu-20.04
```

Comment:
This won't work reliably, as the runner is being decommissioned at the moment. GitHub Actions will commonly just skip that job and then the build will fail as the artifacts are not built. We can catch this and use `ubuntu-latest` for `runs-on` and an appropriate Ubuntu 20.04-based image as the `image`. I can maybe get to implementing this tonight (or otherwise Sunday night).

Reply:
That would be great. Should we maybe move this into a separate PR then? So far it still seemed to run well enough and Maik could use a release created with 20.04 for now. I will probably not be able to work on this until in two weeks.